src/VirtoCommerce.Platform.Core/Security/ApplicationUser.cs

```diff
@@ -15,6 +17,12 @@ public class ApplicationUser : IdentityUser, IEntity
         public virtual string PhotoUrl { get; set; }
         public virtual string UserType { get; set; }
         public virtual string Password { get; set; }
+        public virtual DateTime CreatedDate { get; set; }
+        public virtual DateTime? ModifiedDate { get; set; }
+        [StringLength(64)]
```

Comment:
Remove these DataAnnotations attributes from domain model, they must be defined on the persistent layer for Entity models or on DbContext schema definition.

Reply:
Ok.